Translations/Localization.md

```diff
@@ -0,0 +1,92 @@
+> Localization describes the subsequent process of translating an internationalized plugin. Localization is often abbreviated as l10n (because there are 10 letters between the l and the n.)
```

Comment:
```suggestion
> Localization describes the subsequent process of translating an internationalized theme or plugin. Localization is often abbreviated as l10n (because there are 10 letters between the l and the n.)
```

Reply:
This is a quote from [WordPress handbook](https://developer.wordpress.org/themes/functionality/localization/). Should I change it anyway?